src/services/service-line-template/index.html

```diff
@@ -207,8 +207,21 @@ <h3 class="h4 text-700">{% t software-modernisation.card-4-heading %}</h3>
   </div>
 </section>
 
-<div class=soft-mod__sheen-container>
+{% assign featured_posts =  site.posts | where: 'pinned_locations', "testytest" %}
+{% if featured_posts.size == 1 %}
+  {% assign sheen_container_class = "soft-mod__sheen-container--one-publication" %}
+  {% assign curated_publications_class = "services__curated-publications--one-publication" %}
+{% else %}
+  {% assign sheen_container_class = "soft-mod__sheen-container" %}
+  {% assign curated_publications_class = "services__curated-publications" %}
+{% endif %}
+
+<div class={{sheen_container_class}}>
   {% include compass_teaser.html %}
+
+  {% if featured_posts.size != 0 %}
+    {% include services_curated_publications.html class=curated_publications_class service-line="softmod" tag="software modernisation" pinned_location="testytest" %}
+  {% endif %}
 </div>
```

Comment:
As discussed let's revisit this file together. It's concerning that we're updating the service-line-template and not the soft-mod page. That's due to the repetition that we already have a plan to address, so let's get this into master and when we remove the duplication have another look at how this can be simpified.